padl/transforms.py

```diff
@@ -1623,6 +1623,11 @@ def _pd_splits(self, input_components=0, has_batchify=False) -> Tuple[Union[int,
             else:  # if it's empty: identity
                 final_splits.append(identity)
 
+        if self._pd_name is not None:
```

Comment:
It looks like this code block is the same for the three cases. Could we make it a member function of `Pipeline` or `Transform`?